test/end-to-end/purchases/purchases.spec.js

```diff
@@ -19,11 +20,12 @@ describe('Purchase Orders', () => {
 
   it('supports single item purchase orders', async () => {
     const page = new PurchaseOrderPage();
+    const datePurchase = moment(new Date(), 'YYYY-MM-DD').subtract(1712, 'days');
```

Comment:
By the way, it's helpful to make moment into a date by calling `.toDate()` on the moment object.  Like this:
```js
const datePurchase = moment(new Date(), 'YYYY-MM-DD').subtract(1712, 'days').toDate();
```